certified-connectors/Signi/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2548 @@
+                      "x-ms-summary": "Autosign by proposer"
+                    },
+                    "missing_positions": {
+                      "enum": [
+                        "error",
+                        "append_to_the_end"
+                      ],
+                      "description": "Automatic sign placement",
+                      "type": "string",
+                      "x-ms-summary": "Automatic sign placement"
+                    }
+                  },
+                  "type": "object"
+                },
+                "file": {
+                  "description": "File",
+                  "type": "string",
+                  "default": "uploaded_file_key",
+                  "x-ms-summary": "File",
+                  "x-ms-visibility": "internal",
+                  "title": "File"
+                }
+              },
+              "type": "object",
+              "required": [
+                "fileName",
+                "people",
+                "file",
+                "fileContent"
+              ]
+            },
+            "required": true
+          }
+        ],
+        "summary": "Sign Contract From Provided File (V2)",
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "properties": {
+                "notifications": {
+                  "items": {
+                    "properties": {
+                      "email": {
+                        "description": "Notifications",
+                        "type": "string",
+                        "x-ms-summary": "Notifications"
+                      },
+                      "links": {
+                        "properties": {
+                          "contract": {
+                            "description": "Contract",
+                            "type": "string",
+                            "x-ms-summary": "Contract"
+                          }
+                        },
+                        "type": "object"
+                      }
+                    },
+                    "type": "object"
+                  },
+                  "type": "array"
+                },
+                "state": {
+                  "description": "State of contract",
+                  "type": "string",
+                  "x-ms-summary": "State of contract"
+                },
+                "attachments": {
+                  "items": {
+                    "type": "string"
+                  },
+                  "type": "array"
+                },
+                "contract_id": {
+                  "description": "Contract ID",
+                  "type": "integer",
+                  "x-ms-summary": "Contract ID"
+                }
+              },
+              "type": "object"
+            }
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "operationId": "SignContractFromProvidedFileV2-1",

```

Comment:
@dk-klein all operationIds should be in Camel Case. is it possible to remove the "-" ? Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).
Please fix all operationIds is the swagger file.

Reply:
@Amjed-Ayoub fixed, I removed the "-"